www/docs/server/server-side-calls.md

```diff
@@ -2,7 +2,7 @@
 id: server-side-calls
 title: Server Side Calls
 sidebar_label: Server Side Calls
-slug: /server-side-calls
+slug: /server/server-side-calls
```

Comment:
here we use kebab case (server-side-calls) but in the getQueryKey we used camel case

Reply:
ahhhh I didn't even notice, but you're right, this is a great chance to fix that :)

Reply:
Also all the hooks are useQuery useMutation etc. not sure what the best option is

Reply:
I won't kebab them since it's syntax rather than a phrase, but will lowercase at least just for HTTPness

Reply:
Sounds good

Reply:
links, redirects, and slugs, updated now. I think that should do it :)

Reply:
This probably doesn't matter hugely anyway, the web is case-insensitive, so we can probably keep the internal facing stuff camelCase like the file names and IDs just for readability?

Reply:
Yep - just to calm my ocd xddd